classes/Validator.php

```diff
@@ -67,24 +67,24 @@ public function check(Form $form)
         foreach ($form->getFields() as $field) {
             $name = 'advfrm-' . $field->getName();
             if ($this->isMissing($field)) {
-                $res = $res && $this->checkRequired($form, $field);
+                $res = $this->checkRequired($form, $field);
```

Comment:
That should rather be:
```suggestion
                $res = $this->checkRequired($form, $field) && $res;
```
Analogous for the changes below.

Reply:
Ups, you're right.